apiserver/facades/client/application/application.go

```diff
@@ -58,6 +62,12 @@ var ClassifyDetachedStorage = storagecommon.ClassifyDetachedStorage
 
 var logger = loggo.GetLogger("juju.apiserver.application")
 
+type EcService interface {
```

Comment:
Convention would tend to have this as `ECService`.

Reply:
sure, fixed

Reply:
sure, fixed